### PR TITLE
adding Linalg ones_like

### PR DESCRIPTION
**Pull request by dflorian**
adding Linalg ones_like.

Small initial push for dflorian.

### CODE REVIEW

File: tensornetwork/linalg/linalg.py [hunk 1]
```diff
@@ -149,6 +149,33 @@ def ones(shape: Sequence[int],
   return the_node
 
 
+def ones_like(a: np.ndarray,
```
Comment (Thenerdstation) [inline tensornetwork/linalg/linalg.py:152]: Make this type an Any

Comment (dflorian) [reply tensornetwork/linalg/linalg.py]: changed

**Comment by dflorian** [comment]
@googlebot I signed it!

**Comment by Thenerdstation** [comment]
Looks like theres some formatting errors.
https://travis-ci.org/github/google/TensorNetwork/jobs/701788139

Could you fix these?

**Pull request closed without merging**